session_output example_session.lacp3102.24_09_22_16_53_45_0377 {
session_output example_session.lacp3102.24_09_22_16_53_45_0377 {
	failure Cuda {
		description: <text>Error while calculating cuda: HTTPSConnectionPool(host='null', port=443): Max retries exceeded with url: /vmgr/hmac/vapi/rest/sessions/recalculate-uda-scripts (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f900ce06a58>: Failed to establish a new connection: [Errno -2] Name or service not known'))</text>;
		severity: <text>warning</text>;
		reference: <text>5@/home/lacp3102/APP1/UdeS_S8_APP1/formatif/gei816-formatif-amorce/verif/regression_area/example_session.lacp3102.24_09_22_16_53_45_0377/chain_0/post_session.log</text>;
		failure_source_type_vmgr: <text>post_session</text>;
	}
}
}
session_output example_session.lacp3102.24_09_22_16_53_45_0377 {
	session_status: <text>post_session_script_done</text>;
	session_end_time: <text>1727038447</text>;
	done_chain_id_vmgr: <text>0</text>;
}
